app/domain/authentication/authn_jwt/validate_restrictions_one_to_one.rb

```diff
@@ -0,0 +1,19 @@
+module Authentication
+  module AuthnJwt
+
+    # This class is responsible for retrieving the correct value from the JWT token
+    # of the requested attribute.
+    class ValidateRestrictionsOneToOne
+      def initialize(decoded_token:)
+        @decoded_token = decoded_token
+      end
+
+      def valid_restriction?(restriction)
+        if @decoded_token.key?(restriction.name.to_sym)
+          return @decoded_token[restriction.name.to_sym] == restriction.value
```

Comment:
We do not want to raise an exception if it is not equal ?

Reply:
The command class using it raises exception if its false
https://github.com/cyberark/conjur/blob/e44eaaef953651ebb2458cad6ef69bf6b1a1f040/app/domain/authentication/resource_restrictions/validate_resource_restrictions.rb#L71